Button.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
+
+import threading
+import RPi.GPIO as GPIO
+import time
+import logging
+
+class Button(threading.Thread):
+
+    def __init__(self, channel, timerThread):
+        super(Button, self).__init__()
+        self.channel = channel
+        self.depressed = False
+        self.timer = timerThread
+        self.stop = threading.Event()
+
+    def run(self):
+        GPIO.setmode(GPIO.BCM)
+        GPIO.setup(self.channel, GPIO.IN, pull_up_down=GPIO.PUD_UP)
+        GPIO.add_event_detect(self.channel, GPIO.FALLING, callback=self.edge_callback)
+
+        while True:
```

Comment:
good call. it seemed like the thread was terminating without something keeping it open